modtree/modtree
# fix: improve loading speeds

### Summary of changes

Fixes #293 (and by extension fixes #290)

- Frontend is back to usual fast speed
- Redux user's graphs are now flattened to just ID
- New types `ModtreeApiResponse.UserFull` and `ModtreeApiResponse.GraphFull` (and their empty types) are placeholders to differentiate from the original `.User` and `.Graph` types. Can rename in a future PR

### Testing

With the improved loading speeds, we can consider cypress tests for the following in future. But for now, manual testing.

- Modules done/doing panel (add/remove)
- Edit degree (remove/add modules, change title)
- Add new degree
- Graphs panel shows graphs in our custom format
- Add new graph should show the degree modules info
